content/blog/2018/10/2018-10-14-hackathons-oct.adoc

```diff
@@ -0,0 +1,142 @@
+* Illegal Reflective access patches in EnvInject
+* Testing Platform Labeler Plugin and other plugins with Java 11
+
+After the hackathon we have presented the Java 11 support status at the link:/sigs/platform[Platform SIG meeting] and discussed the next steps
+(link:TODO[meeting notes], link:TODO[video]).
+Generally we are getting close to make Java 11 preview support available in common weekly releases according to the jep:211[] plan.
+All mandatory Jenkins patches have been been already upstreamed to the master branch,
+but we also need to enable CI flows on the master branch before it becomes fully available.
+I plan to make a separate blogpost about it in November.
+
+=== Jenkins Configuration as Code
+
+There was a team working on the plugin:TODO[Configuration-as-Code plugin].
+    link:https://github.com/ewelinawilkosz[Ewelina Wilkosz],
+    link:https://github.com/ndeloof[Nicolas de Loof],
+    link:https://github.com/uhafner[Ulli Hafner],
+    link:https://github.com/martinda[Martin d'Anjou] and
+    link:https://github.com/batmat[Baptiste Mathus]
+focused on this area and did the following:
+
+Accomplishments:
+
+* Some patches for JCasC stability and Web UI
+* Verified support in warnings plugin and added documentation/tests
+* JCasC support in the External Workspace Manager plugin
+* Fixed the random issue in the PR builder
+
+// TODO: Photo
+
+=== Jenkins Infrastructure
+
+Last but not least, Jenkins Infrastructure.
+It was our first time when we had such topic at Jenkins hackathons.
+Olivier Vernin,
+Michael Pailloncy,
+James Albert, and
+Pham Vu Tuan
+worked together in order to improve the project infrastructure.
+
+Summary:
+
+* Jenkins.io and jenkins-ci.org DNS are on Azure
+* Datadog monitoring was automated, repo still needs a transfer
+* Progress on Account app for voting, the app has been also demoed during the hackfest
+* Added JDK11 to the CI infrastructure
+
+//TODO: image
+
+=== Other topics
+
+There were some other activities happening during the hackathon:
+
+* 5 training and demo sessions done by participants:
+    Jenkins Infrastructure,
+    Intro to JCasC development,
+    Warnings Plugin 5.0, etc.
+* Sync-ups on projects: Google Summer of Code 2019, link:TODO
+* Telemetry service ( jep:TODO[]) has been implemented by Daniel Beck
+* Experiments with Pipeline on the K8s storage
+* Mario Kart competitions. We do community bonding, not only the code!
+* Together with Daniel Beck we ran a hand-on Jenkins Weekly release preparation
+
+Full hackfest notes are available link:https://docs.google.com/document/d/1eI3e5u4XoE1q06wILv9sDRvYh91n1JCqZ4M1njcpp_Q/edit#[here].
+
+//TODO: image
+
+== Acknowledgements
+
+I would like to personally thank CloudBees, Alyssa Tong and Liam Newman for making this event happen.
+We did a great progress during the event w.r.t. various topics,
+and it helps to push Jenkins forward.
+
+== Incoming hackathons
```

Comment:
Should move it to a separate blogpost, I'd guess

Reply:
Done.